### ISSUE TITLE
Add keyboard navigation to this sim

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/rutherford-scattering#121]
We would like to add keyboard navigation to this sim. @mbarlow12 would you please work on this one?  When  I use the ?accessibility query param, it looks like only the particle gun is not focusable. Lets try the following focus order to start:

First screen:
- alpha particle gun
- energy slider
- traces check box
- proton/neutron number controls
- scene radio buttons
- play/pause buttons
- reset button

Second screen:
- particle gun
- energy slider
- traces check box
- play/pause buttons
- reset button

**Comment by mbarlow12** [comment]
Focus order and LaserPointer are all working properly. But there's some strange behavior in the sliders. See https://github.com/phetsims/rutherford-scattering/issues/123.

**Comment by mbarlow12** [comment]
Pushed fix for #123. Closing for now.
